package.json

```diff
@@ -25,6 +25,7 @@
     "start": "NODE_ENV=production node server.js",
     "cleandb": "node db/scripts/cleanup_scheduled_calls.js",
     "db:seed": "node db/scripts/seed_database.js",
+    "db:dupeseed": "node db/scripts/seed_same_ward_code_as_admin_code.js",
```

Comment:
might be worth having a slightly more descriptive name for this command?

Reply:
Yeah I agree this wasn't very clear. This has now been removed and into the single migration script for the test